easylinks/src/main/java/com/google/step/servlets/ServletHelper.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
```

Comment:
should be `package com.google.step.servlets;`

Reply:
Fixed！